kolibri/plugins/learn/assets/src/vue/index.vue

```diff
@@ -37,12 +52,44 @@
 
 <script>
 
+  const getters = require('../state/getters');
+  const constants = require('../state/constants');
+  const store = require('../state/store');
+  const PageNames = constants.PageNames;
+
   module.exports = {
+    mixins: [constants], // makes constants available in $options
```

Comment:
Oh, I see how it is getting in now.

Still wondering if it would be better to expose these in as explicit props?